### PR TITLE
updating dask to 2.4.0

### PR DESCRIPTION
**Pull request by stb2145**

### CODE REVIEW

File: base-notebook/binder/environment.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ dependencies:
   - jupyterlab=1
   - jupyterhub=1
   - nbgitpuller=0.7
-  - dask=2.2
+  - dask=2.4.0
   - distributed=2.2
   - dask-kubernetes=0.9
   - dask-gateway=0.2
```
Comment (jhamman) [inline base-notebook/binder/environment.yml]: I think we want to bump all of these:

```yaml
  - dask=2.4
  - distributed=2.4
  - dask-kubernetes=0.9.2
  - dask-gateway=0.3.0

Comment (stb2145) [reply base-notebook/binder/environment.yml]: Gotcha, done!

**Comment by rabernat** [comment]
Shanice, would it also be possible to get xarray updated to 0.13.0 in this PR?

That would involve editing a different file, since xarray is part of `pangeo-notebook` rather than `base-notebook`:
https://github.com/pangeo-data/pangeo-stacks/blob/165039b243b1fc1888c4b48807859329d0be0101/pangeo-notebook/binder/environment.yml#L10

**Comment by stb2145** [comment]
Alright, changed xarray to its newest version 0.13.0 - just waiting on an authorized merge. Thanks!

**Comment by jhamman** [comment]
Thanks @stb2145!

**Comment by rabernat** [comment]
😀

**Pull request merged**